lib/mayaUsd/render/vp2RenderDelegate/mesh.h

```diff
@@ -102,6 +102,8 @@ class HdVP2Mesh final : public HdMesh {
         const HdMeshReprDesc& desc,
         bool requireSmoothNormals, bool requireFlatNormals);
 
+    void _HideAllDrawItems(const TfToken& reprToken);
```

Comment:
We haven't adopted `_` as our naming convention for methods. No need to change it now, but we should stop using it.